docs/config-file-resolutions-yml.md

```diff
@@ -0,0 +1,94 @@
+of a third-party dependency which cannot be fixed or updated.
+
+In such situations, you can *resolve* the error in any future scan by adding a resolution
+to the `resolutions.yml` to mark it as acceptable.
+
+The code below shows the structure of an error resolution in the `resolutions.yml` file:
+
+```yaml
+errors:
+- message: "A regular expression matching the error message."
+  reason: "One of ErrorResolutionReason e.g. BUILD_TOOL_ISSUE,CANT_FIX_ISSUE,SCANNER_ISSUE."
+  comment: "A comment further explaining why the reason above is acceptable."
+```
+where the list of available options for `reason` is defined in
+[ErrorResolutionReason.kt](../model/src/main/kotlin/config/ErrorResolutionReason.kt)
+
+For example, to ignore an error related to a build tool problem, your `resolutions.yml` could include:
+
+```yaml
+errors:
+- message: "Does not have X.*"
+  reason: "BUILD_TOOL_ISSUE"
+  comment: "Error caused by a known issue for which fix is being implemented, see https://github.com/..."
+```
+
+## Resolving Policy Rule Violations
+
+Resolutions should not be not used to resolve license policy violations as they do not change generated open notices.
+To resolve a license policy violation either add a local license finding curation
+to the [.ort.yml file](./config-file-ort-yml.md) if the finding is in your code repository or add a curation to the [curations.yml](config-file-curations-yml.md) if the violation occurs in a third-party dependency.
+
+The code below shows the structure of an policy violation resolution in the `resolutions.yml`file:
+
+```yaml
+rule_violations:
+- message: "A regular expression matching the rule violation message."
+  reason: "One of RuleViolationResolutionReason e.g. CANT_FIX_EXCEPTION, DYNAMIC_LINKAGE_EXCEPTION."
+  comment: "A comment further explaining why the reason above is applicable."
+```
+
+where the list of available options for `reason` is defined in [RuleViolationResolutionReason.kt](../model/src/main/kotlin/config/RuleViolationResolutionReason.kt)
+
+For example, to confirm your organization has acquired an org-wide Qt commercial license, your `resolutions.yml` could include:
+
+```yaml
+errors:
+- message: ".*LicenseRef-scancode-qt-commercial-1.1 found in 'third-party/qt/LICENSE'.*"
+  reason: "LICENSE_ACQUIRED_EXCEPTION"
+  comment: "Org-wide commercial Qt license was purchased, for details see https://jira.example.com/issues/SOURCING-1234"
+```
+
+## Command Line
+
+To use the `resolutions.yml` file pass it to the `--resolutions-file` option of the _reporter_:
+
+```bash
+cli/build/install/ort/bin/ort report
+  -i [evaluator-output-path]/evaluation-result.yml
+  -o [reporter-output-path]
+  --report-formats NoticeByPackage,StaticHtml,WebApp
+  --copyright-garbage-file [ort-configuration-path]/copyright-garbage.yml
+  --resolutions-file [ort-configuration-path]resolutions.yml
+  --license-configuration-file [ort-configuration-path]/licenses.yml
```

Comment:
Remove irrelevant options `--copyright-garbage-file` and `license-configuration-file`.

Reply:
done